### PR TITLE
No Email Support Button When Offline

### PR DESCRIPTION
**Pull request by goldjay**
[FEV-1265](https://auderenow.atlassian.net/browse/FEV-1265)

### CODE REVIEW

**Comment by tpaik18** [comment]
@goldjay Not positive, but it seems like if I'm offline when component is mounting, then my failure to check gift card availability will be considered API_ERROR. I think this should not be considered API_ERROR.

**Comment by goldjay** [comment]
@tpaik18 That's a good point. I'll add a check for connectivity before logging the API_ERROR.

File: FluStudy_au/src/ui/components/GiftCard.tsx [hunk 1]
```diff
@@ -70,10 +70,12 @@ class GiftCard extends Component<Props & WithNamespaces, State> {
         this.setState({ failureReason: response.failureReason! });
       }
     } catch (e) {
-      logFirebaseEvent(AppHealthEvents.GIFTCARD_API_ERROR, {
-        failureReason: GiftcardFailureReason.API_ERROR,
-      });
-      this.setState({ failureReason: GiftcardFailureReason.API_ERROR });
+      if (!isConnected) {
```
Comment (tpaik18) [inline FluStudy_au/src/ui/components/GiftCard.tsx]: Shouldn't this be `if (isConnected)` ?

**Comment by tpaik18** [comment]
Thanks @goldjay. I hope this also resolves the part of FEV-1265 where if I become connected then the screen still shows me "We are experiencing an issue"?

**Pull request merged**